test/lib/types/themeLibrary/ThemeLibraryBuilder.js

```diff
@@ -0,0 +1,46 @@
+const test = require("ava");
+
+const ThemeLibraryBuilder = require("../../../../lib/types/themeLibrary/ThemeLibraryBuilder");
+
+test("tasks", async (t) => {
+	const themeLibraryBuilder = new ThemeLibraryBuilder({
+		resourceCollections: {
+			workspace: {
+				byGlob: async () => {
+					return [];
+				}
+			}
+		},
+		buildContext: {
+			isRootProject: () => {
+				return true;
+			}
+		},
+		project: {
+			metadata: {
+				name: "name",
+				copyright: "copyright"
+			},
+			type: "type"
+		},
+		parentLogger: {
+			createSubLogger: () => {
+				return {
+					createTaskLogger: () => {
+
+					}
+				};
+			}
+		},
+		taskUtil: {
+			isRootProject: () => {
+				return true;
+			}
+		}
+	});
+	const asyncTasks = Object.keys(themeLibraryBuilder.tasks).map((taskKey) => {
+		return themeLibraryBuilder.tasks[taskKey]();
+	});
```

Comment:
Whats the exact purpose of this? Just calling the function and expecting that no error is thrown?

Reply:
Well, better than no test. I've changed the assertion and just called `build` which also invokes the task functions.